lib/glific/partners.ex

```diff
@@ -298,9 +299,37 @@ defmodule Glific.Partners do
 
     organization
     |> Organization.changeset(attrs)
+    |> maybe_pin_newcontact_flow(organization)
     |> Repo.update(skip_organization_id: true)
   end
```

Comment:
```suggestion
    |> Organization.changeset(attrs)
    |> Repo.update(skip_organization_id: true)
    |> pin_newcontact_flow(organization)
  end
```

Reply:
@AkhileshNegi I did not get your code and why we need to move changeset to every function. 
Let me know if the following code make sense to you. 

Also if you were trying to save one extra update query, that I think we can pass because we don't update the organization details two often, infact only one we update the default flow and new contact flow. 

Also, do we need to pin the default flow also ?

Reply:
@pankaj-ag I'm using through changeset we get only those fields which are changed which in case of organization is helpful as we are sending all fields again whenever organization is updated. As we also need to actively update is_pinned field for the new contact flow as well so this way it's easier to check if the flow pinning needs to be updated or not.
Let me know if it makes sense, Happy to find more optimum solutions

Reply:
Yea I don't think we should use a changeset for this check. Either you can use attrs variable to let it be pinned again in case the organization changes the new contact flow and all. 

Also make sure if we are changing the new contact flow it unpinned the last one and pin the new one.

Reply:
@pankaj-ag I have made requested changes